app/src/org/commcare/models/database/user/models/AndroidCaseIndexTable.java

```diff
@@ -279,4 +291,17 @@ public static String getArgumentBasedVariableSet(int number) {
         return sb.toString();
     }
 
+    public void reIndexAllCases(SqlStorage<ACase> caseStorage) {
+        db.beginTransaction();
+        try {
+            wipeTable();
+            for(ACase c : caseStorage) {
+                indexCase(c);
```

Comment:
Yeah, the table is guaranteed to exist. If it's not created yet it gets created in the 5->6 migration. I'll double test against a fresh install as well, for confidence.